account_move_chatter/models/account_move.py

```diff
@@ -1,9 +1,12 @@
+# -*- coding: utf-8 -*-
 # Copyright 2020 Tecnativa - Pedro M. Baeza
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 
-from odoo import models
+from odoo import models, fields
 
 
 class AccountMove(models.Model):
-    _inherit = ["account.move", "mail.thread", "mail.activity.mixin"]
+    _inherit = ["account.move", "mail.thread"]
     _name = "account.move"
+
+    state = fields.Selection(track_visibility='always')
```

Comment:
This is something unrelated  to the purpose of the module and shouldn't be here IMO.

Reply:
Purpose of this change is to track user who is responsible for create, validate, cancel that accounting entry. In v12, this feature covered but in v10, we have to add `track_visibility` attributes. So I did migration and improve little bit.

If you don't like this improvements in a migration PR, I will rollback this changes. And then I will propose a new PR for enhancement. LMK your suggestion.

Thank you.

Reply:
Ok, let's keep it if the final purpose is to include it in the module.